client/src/JobList.test.js

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import { beforeEach, describe, expect, it, jest } from "@jest/globals";
+import { render, shallow, mount } from "enzyme";
+import toJson from "enzyme-to-json";
+import JobList from "./JobList";
+
+const mockJob = jest.fn();
+jest.mock(
+    "./Job",
+    () =>
+        function Job(props) {
+            mockJob(props);
+            return <mock-job />;
+        }
+);
+
+describe("JobList", () => {
+    let props;
+
+    beforeEach(() => {
+        props = {
+            category: "testCategory",
+            data: ["testJob"],
+        };
+    });
+
+    it("renders", () => {
+        const wrapper = render(<JobList {...props} />);
+        expect(toJson(wrapper)).toMatchSnapshot();
+        expect(mockJob).toHaveBeenCalledWith(
+            expect.objectContaining({
+                category: props.category,
```

Comment:
May just be my lack of understanding of how mocking works in Jest, but should we be checking that a proper key property is also passed in here? And where is `children: "testJob"` coming from exactly?

Reply:
It's coming from the mockJob call. It mocks the job component and

Reply:
So mockJob() returns "testJob" because that's what gets passed in to it? And do you think it is worth checking the key property too, or is that not present for some reason?